resources/systemd/openhab2.service

```diff
@@ -6,13 +6,17 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-EnvironmentFile=/etc/default/openhab2
-EnvironmentFile=/etc/default/openhab2_dirs
+Environment=OPENHAB_HOME=/usr/share/openhab2
+Environment=OPENHAB_CONF=/etc/openhab2
+Environment=OPENHAB_RUNTIME=/usr/share/openhab2/runtime
+Environment=OPENHAB_USERDATA=/var/lib/openhab2
+Environment=OPENHAB_LOGDIR=/var/log/openhab2
+Environment=OPENHAB_USER=openhab
+Environment=OPENHAB_GROUP=openhab
+EnvironmentFile=-/etc/default/openhab2
```

Comment:
I see.
So these lines are not actually what the end user is supposed to modify but we will continue to state what was established: modify the default file. At a later point when sysVinit was dropped, we can switch to recommending the service file instead. Is that how you thought of it?